Common/Config/Settings.Designer.cs

```diff
@@ -1,10 +1,10 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by a tool.
-//     Runtime Version:4.0.30319.42000
+//     Ce code a été généré par un outil.
+//     Version du runtime :4.0.30319.42000
```

Comment:
Would be nice to have comments always in english, but probably your windows is in french I suppose and this is aut-generated by VS.

Reply:
Yes of course I never comment in French in my "personal" projects. I can switch my VS to english anyway, as translation isn't that good 😄